README.md

```diff
@@ -233,24 +180,26 @@ application using the same camera.
 The [Media Capture and Streams](https://w3c.github.io/mediacapture-main/)
 specification exposes background blur settings on a `MediaStreamTrack` as
 [capabilities](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediatrackcapabilities-backgroundblur)
-and as a [setting](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediatracksettings-backgroundblur).
+and as a
+[setting](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediatracksettings-backgroundblur).
 
 ### Capabilities
 
 This is a tri-state value that has three possible outcomes:
+
 - `false` means that blur is not currently supported by the platform
 - `true` means that blur is supported and cannot be turned off
 - [`false`, `true`] means that blur is supported, and can be turned on or off
 
-This goes above and beyond what is included in our proposal.  Our proposal does
+This goes above and beyond what is included in our proposal. Our proposal does
 not expose the level of application control for blur state.
 
 We have concerns about whether applications should be allowed to control
 platform blur directly, especially with the current generation of effects
 implementations; in these, changing the effects for one site changes it for
-others that are also consuming the same camera stream. 
+others that are also consuming the same camera stream.
 
-Also, we are considering extensions to this proposal.  These extensions include:
+Also, we are considering extensions to this proposal. These extensions include:
 
 - An event that would let applications know if blur support has changed
   dynamically;
```

Comment:
A `configurationchange` event is fired on the track when blur state changes, so we don't need a track level event.  I don't think an event is needed on `VideoFrameMetadata` as the application will get the true state with every frame and can keep track of changes itself.
```suggestion
```